src/data/zmod/basic.lean

```diff
@@ -128,7 +128,10 @@ def cast : Π {n : ℕ}, zmod n → R
 @[priority 900] instance (n : ℕ) : has_coe_t (zmod n) R := ⟨cast⟩
 
 @[simp] lemma cast_zero : ((0 : zmod n) : R) = 0 :=
-by cases n; simp
+-- `zmod 0` unifies with `ℤ` but doesn't inherit the `coe_is_add_monoid_hom` instance
+-- so we add `int.cast_zero` to the `simp` set despite being an instance of the
+-- existing `@[simp] lemma coe_zero`
+by cases n; simp [int.cast_zero]
```

Comment:
I don't fully understand the explanation here; can't we add a `coe_is_add_monoid_hom` instance for `zmod 0` by copying the int one?